Makefile

```diff
@@ -4,9 +4,8 @@ NUM_THEMES=$(NUM_ADDONS)
 
 UNAME_S := $(shell uname -s)
 
-# If you're using docker and docker-compose, you can use this Makefile to run
-# commands in your docker images by setting the DOCKER_PREFIX environment variable
-# to: DOCKER_PREFIX="docker-compose run --rm web"
+ # If you're using docker and docker-compose, you can use this Makefile to run
+ # commands in your docker images by executing `docker-compose run --rm web make <target>`
```

Comment:
This should refer to using `docker exec` to get a shell on the running container first:

```
docker exec -t -i [container-id] /bin/bash
```
This is neatly wrapped up if you use docker-utils [1] (this is just a wrapper for docker-compose). Then it should be just a case of `docker-utils bash web` and run any make commands you want.

This will save spinning up new images just to run a command when you've probably got it running already.
 
[1] https://pypi.python.org/pypi/docker-utils